docs/vocabulary.md

```diff
@@ -8,27 +8,117 @@ hide_title: true
-| C19HCC Confirmed COVID19 Infection SNOMEDCT                      | SNOMEDCT          | Extensional | [2.16.840.1.113762.1.4.1032.114](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.114/expansion/20200410) | 1  |
-| C19HCC Hydroxychloroquine                                        | RXNORM            | Extensional | [2.16.840.1.113762.1.4.1032.127](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.127/expansion/20200417) | 12 |
-| C19HCC SARS coronavirus 2 Qualitative Detection Laboratory Tests | LOINC             | Extensional | [2.16.840.1.113762.1.4.1032.109](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.109/expansion/20200417) | 0  |
-| C19HCC SARS related Qualitative Detection Laboratory Tests       | LOINC             | Extensional | [2.16.840.1.113762.1.4.1032.113](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.113/expansion/20200410) | 0  |
-| C19HCC SARSCoV2 Exposure                                         | ICD10CM, SNOMEDCT | Grouping    | [2.16.840.1.113762.1.4.1032.120](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.120/expansion/20200410) | 2  |
-| C19HCC SARSCoV2 Exposure ICD10CM                                 | ICD10CM           | Extensional | [2.16.840.1.113762.1.4.1032.118](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.118/expansion/20200410) | 1  |
-| C19HCC SARSCoV2 Exposure SNOMEDCT                                | SNOMEDCT          | Extensional | [2.16.840.1.113762.1.4.1032.119](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.119/expansion/20200410) | 1  |
-| C19HCC Suspected COVID19 Infection                               | ICD10CM, SNOMEDCT | Grouping    | [2.16.840.1.113762.1.4.1032.116](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.116/expansion/20200410) | 14 |
-| C19HCC Suspected COVID19 Infection ICD10CM                       | ICD10CM           | Extensional | [2.16.840.1.113762.1.4.1032.106](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.106/expansion/20200410) | 13 |
-| C19HCC Suspected COVID19 Infection SNOMEDCT                      | SNOMEDCT          | Extensional | [2.16.840.1.113762.1.4.1032.115](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.115/expansion/20200410) | 1  |
-
-Coming Soon:
-
-* Respiratory Condition: ICD-10-CM J00-J99
-* Invasive mechanical ventilation: Any intubation procedure or mechanical ventilation-associated procedure, excluding supplemental oxygen (high or low flow), CPAP, and BiPAP, ECMO (SNOMED CT, ICD-10-CM).
+Value sets supporting cohort definitions. Listing of value sets is in alphabetical order. Some value sets refer to an outside source or repository, and others are expanded here.
+
+### BCG Vaccine Products for Intravesical Use​ VS
```

Comment:
I think this is fine for now - but I need to take an action to create these in VSAC.

Reply:
Yes, or reference another existing source.